modules/version.js

```diff
@@ -0,0 +1,20 @@
+var exec = require('child_process').exec;
+
+var hash;
+
+module.exports.init = function(b) {
+	exec('git rev-parse HEAD', function(error, stdout, stderr) {
```

Comment:
`child_process` is part of the npm stdlib; do I really need a package.json for that?